packages/eslint-plugin/docs/rules/consistent-type-imports.md

```diff
@@ -36,6 +36,49 @@ type T = Foo;
 const x: Bar = 1;
 ```
 
+### `fixStyle`
+
+This option defines the expected type modifier to be added when an import is detected as used only in the type position. Valid values for `fixStyle` are:
+
+- `separate-type-imports` will add the type keyword after the import keyword `import type { A } from '...'`. It is the default.
+- `inline-type-imports` will inline the type keyword `import { type A } from '...'` and is only available in TypeScript 4.5 and onwards. See [documentation here](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-4-5.html#type-modifiers-on-import-names).
+
+Examples of the fixer with `{ fixStyle: 'separate-type-imports' }`:
```

Comment:
Docs nit: I think you could streamline this explanation with tabs.

```md
<!--tabs-->

#### ❌ Incorrect

(original code snippet)

#### ✅ With `inline-type-imports`

(separate-type-imports fixed version)

#### ✅ With `separate-type-imports`

(separate-type-imports fixed version)

<!-- /tabs -->
```

What do you think?

Reply:
This is how it looks now!

<img width="537" alt="Screen Shot 2022-11-07 at 4 40 37 PM" src="https://user-images.githubusercontent.com/7374640/200430731-be97fbbb-a63c-4c84-9ed2-f8c91ea1fca0.png">

Reply:
Close enough 😄